routes/cartRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const cartController = require("../controllers/cartController");
+const { loginRequired } = require("../utils/auth");
+const router = express.Router();
+
+router.get("", loginRequired, cartController.cartInfo);
+router.get("/delete/:productId", loginRequired, cartController.deleteInCart);
+router.get("/deleteAll", loginRequired, cartController.deleteAllInCart);
```

Comment:
- url에는 동사를 사용하지 않습니다.
- delete method를 사용해 주세요.
- cart/:productId라고 했는데, RESTful에 의하면 cart 뒤에 오는 숫자는 product의 ID가 아닌 cart의 ID로 인식하가 쉽습니다. 따라서 params로 받으시려면 cartId를 받아주시고, productId를 받고 싶은면, query로 받아주세요.
```suggestion
router.delete("/:cartId", loginRequired, cartController.deleteInCart);
router.delete("", loginRequired, cartController.deleteAllInCart);
```